CHANGELOG.md

```diff
@@ -24,6 +24,16 @@ so they can be moved to a respective version upon release.
 
 Add your changes here.
 
+## [2.1.0] - 2020-02-24
+
+- Fine tune API changes ([#216](https://github.com/airsidemobile/JOSESwift/pull/216)) via [@daniel-mohemian](https://github.com/daniel-mohemian)
```

Comment:
For a changelog I'd suggest being more descriptive: what does "fine tune API changes" mean for the end user?